src/dsl.jl

```diff
@@ -317,7 +323,7 @@ function make_reaction_system(ex::Expr; name = :(gensym(:ReactionSystem)))
 
     # Reads equations. 
     vars_extracted, add_default_diff, equations = read_equations_options(
-        options, variables_declared)
+        options, variables_declared; noinfer = noinfer)
```

Comment:
```suggestion
        options, variables_declared; noinfer)
```

Reply:
Does this actually work? We need the variable name since it's a kwarg right?

Reply:
If a kwarg is the same as the variable name you don’t need to repeat it (when you use “;”).